src/client/pages/Docs.mad

```diff
@@ -9,7 +9,6 @@ import {
 import { strata } from "Strata"
 
 import { Nav } from "@client/views/Nav"
-import { Header } from "@client/views/Header"
```

Comment:
Great move, I just we might want to do this for `Nav` and `Footer` as well then.